workers/unity/Packages/com.improbable.gdk.core/Systems/CleanReactiveComponentsSystem.cs

```diff
@@ -87,6 +87,21 @@ private void AddRemoveComponentAction<T>()
                 }
             });
         }
+        
+        public void UsedOnlyForAotCodeGeneration()
+        {
+            // IL2CPP needs those lines for AOT instantiation of methods
+            AddRemoveComponentAction<OnConnected>();
```

Comment:
This is error-prone if we need to instantiate every method that AOT needs. Unfortunately, IL2CPP can only be used with AOT, which in turn, needs instantiation of these methods.
I'll try to experiment to see what we can do here to make this more